install/update_0831_084.php

```diff
@@ -1913,15 +2199,25 @@ function updateNetworkFramework(&$ADDTODISPLAYPREF) {
       $DB->queryOrDie($query, "0.84 create glpi_ipnetworks");
 
       // Retrieve all the networks from the current network ports and add them to the IPNetworks
-      $query = "SELECT DISTINCTROW INET_NTOA(INET_ATON(`ip`)&INET_ATON(`netmask`)) AS address,
-                     `netmask`, `gateway`, `entities_id`
-                FROM `origin_glpi_networkports`
-                ORDER BY `gateway` DESC";
+      $iterator = $DB->request([
+         'SELECT'    => [
+            new \QueryExpression(
+               "INET_NTOA(INET_ATON(" . DBmysql::quoteName("ip") . ")&INET_ATON(" .
+               DBmysql::quoteName("netmask") . ")) AS " . DBmysql::quoteName("address")
+            ),
+            "netmask",
+            "gateway",
+            "entities_id"
+         ],
+         'DISTINCT'  => true,
+         'FROM'      => "origin_glpi_networkports",
+         'ORDER'     => "gateway DESC"
+      ]);
       $address = new IPAddress();
```

Comment:
Not mandatory, but this variable is not used (redefined later for another usage).
```suggestion
```